src/index.html

```diff
@@ -8,5 +8,39 @@
   </head>
   <body>
     <h1>Calendar</h1>
+
+    <div class="calendar calendar--start-day-ppp calendar--month-length-21">
```

Comment:
add normal default values

```suggestion
    <div class="calendar calendar--start-day-wed calendar--month-length-31">
```